src/transports/smart_protocol.c

```diff
@@ -19,6 +19,8 @@
 /* The minimal interval between progress updates (in seconds). */
 #define MIN_PROGRESS_UPDATE_INTERVAL 0.5
 
+bool git_ofs_delta__enabled = true;
```

Comment:
ok, I was not aware of the naming scheme, will update it to `git_smart__ofs_delta_enabled`